client/src/main/java/io/pravega/client/stream/Stream.java

```diff
@@ -42,4 +44,17 @@
      */
     String getScopedName();
 
+    /**
+     * Helper utility to create a Stream object.
+     *
+     * @param scope Scope of the stream.
+     * @param streamName Name of the stream.
+     * @return Stream.
+     */
+    static Stream of(String scope, String streamName) {
```

Comment:
Is this to be used by the user, where and how?   Should the `ReaderGroupConfigBuilder.stream` method use this class rather than a `String`?

Reply:
Based on the discussion added two helper methods.

a. `Stream of(String scope, String streamName)`
b. `Stream of(final String scopedName)`